### ISSUE TITLE
Running project (Documentation)

### ISSUE THREAD

**Issue by OmkarMozar** [philipphager/pscout-legacy-parser#1]
@philipphager I appreciate your work for this .
Can you help me to run this project?

**Comment by philipphager** [comment]
Sure, sorry for being not more proactive at the moment, and for the lack of documentation, will fix that in the next days.

For now you can build the project like this:
1.) Clone / Download the project
2.) `cd` into the project directory
3.) Run `./gradlew assemble` to build the project. The executable jar will be located in `/build/libs`
4.) Run it with `java -jar ./build/libs/pscout-parser-0.1.jar -i path/to/your/pscout1/file.txt -o /path/to/the/output/directory`. Just provide the parameters -i for the pscout input file and -o for the output directory for the parsed .csv file

**Comment by philipphager** [comment]
@OmkarMozar I also just published the current jar file [here](https://github.com/philipphager/pscout-legacy-parser/releases/tag/v0.1). So if you want to, you can just download the .jar file and run `java -jar pscout-parser-0.1.jar -i path/to/your/pscout1/file.txt -o /path/to/the/output/directory`

**Comment by philipphager** [comment]
@OmkarMozar let me know, if this worked for you, or if you need any help :)

**Comment by OmkarMozar** [comment]
Yes, It worked.. :) Thank you for your help. Just make sure you update csv file for Undocumented as well as documented apis ,then it must be of size 4 mb for api-22 not 3.7 mb . Just cross check it.
